runtime/j9vm/java11vmi.c

```diff
@@ -1634,58 +1634,57 @@ JVM_SetBootLoaderUnnamedModule(JNIEnv *env, jobject module)
 	vmFuncs->internalEnterVMFromJNI(currentThread);
 	if (NULL == module) {
 		vmFuncs->setCurrentExceptionUTF(currentThread, J9VMCONSTANTPOOL_JAVALANGNULLPOINTEREXCEPTION, "module is null");
-	} else {
-		j9object_t modObj = J9_JNI_UNWRAP_REFERENCE(module);
+	} else if (NULL == currentThread->currentException) {
 		J9ClassLoader *systemClassLoader = vm->systemClassLoader;
+		J9Class *moduleClass = vmFuncs->internalFindKnownClass(currentThread,
```

Comment:
some C compilers complain about declaring a variable after calling a function. Does this compile on windows?

Reply:
[JDK11_x86-64_windows](https://hyc-runtimes-jenkins.swg-devops.com/job/Build_JDK11_x86-64_windows_Personal/1818/) finished successfully.